pageserver/src/tenant/mgr.rs

```diff
@@ -980,7 +980,7 @@ impl TenantManager {
                 // Directory doesn't need to be fsync'd because we do not depend on
                 // it to exist after crashes: it may be recreated when tenant is
                 // re-attached, see https://github.com/neondatabase/neon/issues/5550
-                tokio::fs::create_dir_all(&tenant_path)
+                tokio::fs::create_dir_all(&timelines_path)
```

Comment:
I think this also needs to be changed for the `Secondary` case above. We don't really need to duplicate directory creation either, we can just do it above the match.

Reply:
For the secondary case, the downloader code (not merged yet) did a create_dir_all for each timeline, so didn't care about existence of timeline path.  However, that's a bit subtle, so yeah: let's pull this up to be common for both cases.  The config write was also the same for both cases.  b00d199ad